### ISSUE TITLE
Expand gufo:isDerivedFrom domain and range?

### ISSUE THREAD

**Issue by tgoprince** [nemo-ufes/gufo#3]
The object property `gufo:isDerivedFrom` has `gufo:EndurantType` as its range.

That prevents us from representing that a historical relation is derived from an event.

For instance:

```
:isAuthorOf rdf:type owl:ObjectProperty;
    rdfs:domain :Person;
    rdfs:range :Book;
    gufo:isDerivedFrom :CompositionProcess .

:CompositionProcess rdfs:subClassOf gufo:Event .
```

**Comment by tgoprince** [comment]
What do you think @ claudenirmf?

**Comment by claudenirmf** [comment]
I agree with you, we should open the range to include events to support historical relations. We should also check if it is ok to "bypass" the endurant in other to have the "strong truthmaking pattern" without the "weak truthmaking pattern".
